src/site/content/en/blog/model-viewer/index.md

```diff
@@ -0,0 +1,160 @@
+to a web page, while hosting the model on your own site. The web component
+supports responsive design and use cases like augmented reality on some devices,
+and it includes features for accessibility, rendering quality, and
+interactivity.  The goal of the component is to enabble adding 3D models to your
+website without understanding the underlying technology and platforms.
+
+{% Aside %}
+We're always [updating and
+improving](https://github.com/google/model-viewer/releases)
+`<model-viewer>`. Check out the [`<model-viewer>`
+homepage](https://modelviewer.dev/) to explore what &lt;model-viewer&gt; 1.0 can do.
+{% endAside %}
+
+## What is a web component?
+
+A web component is a custom HTML element built from standard web platform
+features. A web component behaves for all intents and purposes like a standard
+element. It has a unique tag, it can have properties and methods, and it can
+fire and respond to events. In short, you don't need to know anything special to
+use any web component, much less `<model-viewer>`.
+
+In this article, I will show you things that are particular to `<model-viewer>`.
+If you're interested in learning more about web components
+[webcomponents.org](https://www.webcomponents.org/) is a good place to start.
+
+## What can &lt;model-viewer> do?
+
+I'll show you a few current capabilities of `<model-viewer>`. You'll get a great
+experience today, and `<model-viewer>` will get better over time as Google adds
+new features and improves rendering quality. The examples I'm provided are just
+to give you a sense of what it does. If you want to try them there are
+installation and usage instructions in [its GitHub
+repo](https://modelviewer.dev/).
+
+### Basic 3D models
+
+Embedding a 3D model is as simple as the markup below. By
+using glb files, we've ensured that this component will work on any major
+browser.
+
+```html
+<!-- Import the component -->
+<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
+<script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
+<!-- Use it like any other HTML element -->
+<model-viewer id="mv-demo" shadow-intensity="1" src="./spacesuit.glb"
+alt="A 3D model of an astronaut" auto-rotate camera-controls
+poster="./spacesuit.jpg"></model-viewer>
+```
+
+That code renders like this:
+
+<div class="glitch-embed-wrap" style="height: 500px; width: 100%;">
+  <iframe
+    src="https://model-viewer-shark.glitch.me/"
+    title="<model-viewer> on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media;
+clipboard; clipboard-read; clipboard-write"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+### Adding motion and interactivity
+
+The `auto-rotate` and `camera-controls` attributes provide motion and user
+control. Those aren't the only possible attributes. See the documentation for [a
+complete list of](https://modelviewer.dev/).
```

Comment:
```suggestion
complete list of attributes](https://modelviewer.dev/#section-attributes).
```
You were missing a word and we should link them directly to the attributes documentation